.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - "7"
+  - "8"
+cache:
+  directories:
+    - node_modules
+script:
+  - cd example && npm install && npm run build
```

Comment:
Could you remove `script` and replace `prepublish` npm script with `prepare`? travis by default runs `npm install` + `npm test` for `node_js` language.